kudu/src/main/java/org/apache/apex/malhar/kudu/sqlparser/KuduSQLExpressionLexer.java

```diff
@@ -0,0 +1,212 @@
+/**
```

Comment:
Why check in the generated files?

Reply:
Thanks for raising this. I initially had to check them in as the IDE was not recognising the automatically generated sources. This caused error markers in the IDE until there was a maven build done properly. Removed all of auto generated files from the source tree now.

Reply:
@tweise / @vrozov  - The build is failing on Travis because the auto-generated files do not have the license headers. The license check is failing the build. Please advise if there is an approach that is followed in such situations.

Reply:
Is the solution to generate the code under the target folder and not under src? That may also eliminate the need to do special things for gitignore or checkstyle.

Reply:
Thanks for the suggestion. This is a good idea. I will explore to see how the Antlr4 plugin can be coerced to do this. I will reply as a comment as soon as I am done with this experiment.